### ISSUE TITLE
Add "Uninstall ESM-tools" section to Doc

### ISSUE THREAD

**Issue by SpontEIN** [esm-tools/esm_tools#73]
Add "Uninstall ESM-tools" section to Doc:

something like:

### Unix-based System:
To uninstall and remove your current installation of ESM-tools, you need to remove the installed python libs and deltete the esm_* executables. Following commands will do the trick: 

```
rm ~/.local/bin/esm*

rm ~/.local/lib/python3.6/site-packages/esm*

rm ~/.local/lib/python3.6/site-packages/easy-* 
```

**Comment by pgierz** [comment]
Sure, you can put that as a new section here:

https://github.com/esm-tools/esm_tools/blob/release/docs/installation.rst

At some point it'll be nice to move away from the `install.sh` and instead use regular pip to install everything, but that still needs a little bit of work; in particular how the configs are read (currently, they rely on FUNCTION_PATH in the rc file.) I have a trick for the configs, but it isn't clear how to easily set and retrieve user information.

**Comment by SpontEIN** [comment]
Can I simply commit to the repository?
(Btw.: I am Daniel working at AWI Potsdam)

**Comment by pgierz** [comment]
No, you need to make a fork and then submit a pull request and either myself or one of the other developers will put it in. There should be some instructions in the user manual. Just click the big "fork" button in the top right of GitHub and then you get your own copy you can work with, commit to, and submit pull requests back to us.

**Comment by pgierz** [comment]
Out of curiosity, I am guessing you want to uninstall or reinstall because something went wrong: what went wrong?

**Comment by SpontEIN** [comment]
> Out of curiosity, I am guessing you want to uninstall or reinstall because something went wrong: what went wrong?

Dirk already helped me with the issue. The problem did not become fully clear. But the reinstall helped.
I think it had to do with a previous installation, potentially of an older version of the esm-tools. I already installed the tools some months ago, then forgot about it again (in between change to new github repo) ... so I think it was a problem on my side.

**Comment by pgierz** [comment]
Merged in #74, closing.
